database/tables/workspaces.pg

```diff
@@ -0,0 +1,17 @@
+columns
+    id: bigint not null default nextval('workspaces_id_seq'::regclass)
+    s3_bucket: text
+    s3_root_key: text
+    state: text
```

Comment:
Maybe we actually should use an enum here.

Reply:
Added as `enum_workspace_state`